src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -40,6 +41,7 @@
     public static final String VALID_GENDER_BOB = "M";
     public static final String VALID_TAG_HUSBAND = "husband";
     public static final String VALID_TAG_FRIEND = "friend";
+    public static final String VALID_BLOCKROOM = "A420";
```

Comment:
perhaps you can split this up to VALID_BLOCK and VALID_ROOM, so that you can reuse them? (especially in TypicalPersons.java)

You can make VALID_BLOCKROOM with the 2 as well.

Reply:
They are regarded as 1 entry in the add command though